src/lib/search-and-replace.ts

```diff
@@ -130,7 +123,7 @@ export const searchAndReplace = async (
 	const fileSize = getFileSize( fileName );
 
 	// if we don't have any pairs to replace with, return the input file
-	if ( ! pairs || ! pairs.length ) {
+	if ( ! pairs.length ) {
```

Comment:
`pairs: string | string[]`. Luckily, both `String` and `Array` have `.length` :-)